en-US/firefox/whatsnew_69.lang

```diff
@@ -0,0 +1,64 @@
+## NOTE: Demo page at https://www-demo1.allizom.org/firefox/69.0/whatsnew/
+## URL: https://www-demo1.allizom.org/%LOCALE%/firefox/69.0/whatsnew/
+
+
+# HTML title. Firefox Monitor should remain in English as a product name.
+;Firefox Monitor — see if you’ve been in another company’s data breach
+Firefox Monitor — see if you’ve been in another company’s data breach
+
+
+# HTML description. Firefox Monitor should remain in English as a product name.
+;Use Firefox Monitor to see if your info was compromised in another company’s data breach — and get automatically signed up for future alerts.
+Use Firefox Monitor to see if your info was compromised in another company’s data breach — and get automatically signed up for future alerts.
+
+
+# Notification banner
+;Congrats! You’re using the latest version of Firefox.
+Congrats! You’re using the latest version of Firefox.
+
+
+# Page headline for users without an account. Firefox Monitor should remain in English as a product name.
+;Have you checked out Firefox Monitor?
+Have you checked out Firefox Monitor?
+
+
+;Sign up to see if your info was compromised in another company’s data breach, and automatically get alerts when your info might be at risk.
+Sign up to see if your info was compromised in another company’s data breach, and automatically get alerts when your info might be at risk.
+
+
+# Monitor refers to the Firefox Monitor product. It should remain in English.
+;Sign Up for Monitor
+Sign Up for Monitor
+
+
+# Page headline for account holders. Firefox Monitor should remain in English as a product name.
+;Have you checked out Firefox Monitor? It’s included with your Firefox account.
+Have you checked out Firefox Monitor? It’s included with your Firefox account.
+
+
+;Sign In
+Sign In
+
+
+;Your Firefox Browser now automatically blocks online snoopers
+Your Firefox Browser now automatically blocks online snoopers
+
+
+;Now it’s harder than ever for spying companies to track you online. That’s because Enhanced Tracking Protection — a feature that blocks known tracking cookies — is on by default in your updated browser.
+Now it’s harder than ever for spying companies to track you online. That’s because Enhanced Tracking Protection — a feature that blocks known tracking cookies — is on by default in your updated browser.
+
+
```

Comment:
Add a comment that Pocket is a product name and should remain in English. Pocket is not offered in most of the markets that the mozorg site is localized in.